crates/electrum/src/electrum_ext.rs

```diff
@@ -46,8 +46,10 @@ impl RelevantTxids {
         let new_txs = client.batch_transaction_get(&missing)?;
         let mut graph = TxGraph::<ConfirmationHeightAnchor>::new(new_txs);
         for (txid, anchors) in self.0 {
-            if let Some(seen_at) = seen_at {
-                let _ = graph.insert_seen_at(txid, seen_at);
+            if anchors.is_empty() {
+                if let Some(seen_at) = seen_at {
```

Comment:
❓ I know it breaks the current API but shouldn't `seen_at` be required and non-optional here too?

Reply:
I don't know whether it makes more sense to make this `seen_at` parameter required, or to remove the param and use the new `update_last_seen_unconfirmed` method on the update `TxGraph`.

Reply:
I think it's fine as it is. Will likely be radically changed soon.